bonus.md

```diff
@@ -3,11 +3,20 @@
 Edit this file to answer the following questions. Give examples. You may need to do a bit of research. :)
 
 1. Why is Hash.fetch useful?
+It allows you to set a default value
```

Comment:
What is the difference in behavior of `args[:key]` vs `args.fetch(:key)` when `:key` is not present in the hash?